db/schema.rb

```diff
@@ -10,11 +10,27 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_06_29_134710) do
+ActiveRecord::Schema.define(version: 2020_06_30_120116) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "groups", force: :cascade do |t|
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.string "class_name"
+    t.string "#<ActiveRecord::ConnectionAdapters::PostgreSQL::TableDefinition"
```

Comment:
what's this? something went wrong

Reply:
I see you made models for student, student_group and group, but you only made migrations for student and group. I think this is the origin of this strange last line

Reply:
yes that's probably what happen, should we delete this line or make a new migration for the  groups